website/docs/docs/collaborate/column-level-lineage.md

```diff
@@ -0,0 +1,58 @@
+---
+
+dbt Explorer now offers column level lineage (CLL) for the resources in your dbt project. Analytics engineers can quickly and easily gain insight into the provenance of their data products at a more granular level. For each column in a dbt resource (model, source, or snapshot), Explorer provides the full lineage for the data in that column.
+
+Column level lineage is available to dbt Cloud Enterprise accounts that can use Explorer. It’s also available through the Discovery API.
+
+:::tip Beta
+Column-level lineage is now available in beta. Check it out! We'd love to [know what you think](https://docs.google.com/forms/d/e/1FAIpQLSdpCbVkGY9QwfExFonpWE4DTOKi3fQxBGLD0wwKYpkMjgcE7g/viewform)!
+:::
+
+## No setup required
+
+There is no additional setup required for column level lineage if your account is on an Enterprise plan that can use Explorer. You can access column level lineage by expanding the column card in the **Columns** tab of an Explorer [resource details page](/docs/collaborate/explore-projects#view-resource-details) for a model, source, or snapshot.
+
+Lineage will update after each run executed in the production environment. Make sure that `docs generate` is running within at least one job in the environment. Refer to [Generating metadata](/docs/collaborate/explore-projects#generate-metadata) for more details.
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/example-cll.png" title="Example of the Columns tab and where to expand for the CLL"/>
+
+<LoomVideo id='278c948ba387457884cc6b9545793685' />
+
+## What you can use column level lineage for {#use-cases} 
+
+Click on these tabs to learn more about the CLL use cases, the analysis you can do, and the results you can achieve:
+
+<Tabs>
+<TabItem value="root-cause" label="Root cause analysis">
+
+When there is an unexpected breakage in a data pipeline, column level lineage can be a valuable tool to understand the exact point in the pipeline where the error took place. For example, a failing data test on a particular column in your dbt model might've stemmed from an untested column upstream. Using CLL can help quickly identify and fix breakages when they happen.
+
+</TabItem>
+<TabItem value="impact" label="Impact analysis">
+
+During development, analytics engineers can use column level lineage to understand the full scope of the impact of their proposed changes. This knowledge empowers them to create higher quality pull requests that require less rework, as they can anticipate and preempt issues that would've been unchecked without column level insights. 
+
+</TabItem>
+<TabItem value="collaboration" label="Collaboration and efficiency">
+
+When exploring your data products, navigating column lineage allows analytics engineers and data analysts to more easily navigate and understand the origin and usage of their data, enabling them to make better decisions with higher confidence.
+</TabItem>
```

Comment:
Another use case - thoughts @dave-connors-3 ?

**Auditing**

Column-level awareness makes it easier for data developers to see how columns flow through the project when examining how data is used in a dbt project.